test/components/DayPicker_spec.jsx

```diff
@@ -260,6 +261,20 @@ describe('DayPicker', () => {
       });
     });
 
+    describe('#componentWillReceiveProps', () => {
```

Comment:
This test can't run currently because of https://github.com/airbnb/react-dates/blob/master/test/components/DayPicker_spec.jsx#L225

Is there an open issue/discussion for how to solve? getting `It looks like you called `mount()` without a global document being loaded.` when i remove `skip`